### DISCUSSION TITLE
Bem-vindo(a) ao Discussions do SAF-T-AO!

### DISCUSSION THREAD

**Discussion by cryptolopes** [assoft-portugal/SAF-T-AO#109] [General]
## 👋 Bem-vindo!

Estamos a usar o [Discussions](https://github.com/assoft-portugal/SAF-T-AO/discussions) como forma de facilitar o diálogo entre os membros da comunidade. Esperamos que:
- Faça perguntas sobre as quais está a pensar.
- Partilhar ideias.
- Envolva-se com outros membros da comunidade.
- Encorage outros a participar! Lembre-se que esta é uma comunidade que estamos a construir juntos 💪.

Para começar, propomos que comente abaixo, fazendo uma apresentação sobre si mesmo e contando-nos a sua experiência com o SAF-T(AO).

**Comment by andre930-amta** [comment]
Olá Pessoal estou com este problema no portal da AGT:

Falha na validação do campo AuditFile.Header.ProductID

Ajuda please

**Comment by andre930-amta** [comment]
Olá Pessoal estou com este problema no portal da AGT:

Falha na validação do campo AuditFile.Header.ProductID

Ajuda please
